servers/relay/test/http.spec.ts

```diff
@@ -1,11 +1,11 @@
-import "mocha";
 import axios, { AxiosInstance } from "axios";
-import { use, expect } from "chai";
+import { expect, use } from "chai";
 import chaiHttp from "chai-http";
 import { Agent } from "https";
-
+import "mocha";
```

Comment:
This seems like an unrelated change, was this intentional?

Reply:
prettier slipped through when I visited this file